client/rpc/root.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // Register REST endpoints
-func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router) {
+func RegisterRPCRoutes(cliCtx context.CLIContext, r *mux.Router) {
```

Comment:
We shouldn't have done this tbh. The package is self describing.

Reply:
This is a downside of aliasing sub packages functions at top level: you need to come up with unique names at every turn. I merged it anyway because it was consistent with @rigelrozanski's view on this.